src/test/java/games/strategy/ui/SwingWorkerCompletionWaiterTest.java

```diff
@@ -1,24 +1,20 @@
 package games.strategy.ui;
 
+import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
 
 import java.beans.PropertyChangeEvent;
 
 import javax.swing.SwingWorker;
 
 import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.mockito.InjectMocks;
-import org.mockito.Mock;
-import org.mockito.junit.MockitoJUnitRunner;
 
-@RunWith(MockitoJUnitRunner.class)
 public final class SwingWorkerCompletionWaiterTest {
```

Comment:
I reverted the changes to this file on your branch and was able to run the test successfully within Eclipse.  I even changed it to use the `StrictStubs` test runner without a problem.  Before we start getting rid of what's pretty standard Mockito boilerplate, we should probably spend a bit of time trying to determine the root cause of the problem on your machine.  Can you please post the error you're getting when running this test in Eclipse?

(Just thought of something... since you were playing around with Java 9 in the last 24 hours, might you still be building with JDK 9 in Eclipse?)